cypress/integration/ShowCertification.js

```diff
@@ -45,7 +45,7 @@ describe('A certification,', function () {
         .click();
       cy.contains('Submit and go to next challenge').click().wait(1000);
     });
-    cy.get('.react-monaco-editor-container', { timeout: 60000 });
+    cy.get('.donation-modal').should('be.visible');
```

Comment:
I spent way too long wondering about why this test failed, because it is odd. Consider removing?

Reply:
We're still expecting this to happen, aren't we?

Reply:
Yes, it should still happen. It is just not part of what is being tested.